Upstream: Not applicable
Reason: Remove command not meant for packagers

diff --git a/install.sh b/install.sh
index e032fb2..4b0e5e0 100755
--- a/install.sh
+++ b/install.sh
@@ -107,8 +107,6 @@ install_theme() {
   ln -sr "${THEME_DIR}/24"                                                       "${THEME_DIR}/24@2x"
   ln -sr "${THEME_DIR}/32"                                                       "${THEME_DIR}/32@2x"
   ln -sr "${THEME_DIR}/scalable"                                                 "${THEME_DIR}/scalable@2x"
-
-  gtk-update-icon-cache "${THEME_DIR}"
 }
 
 while [ $# -gt 0 ]; do
